README.md

```diff
@@ -94,7 +94,7 @@ Override Sylius template by create a new file in the folder: `templates/bundles/
 
         # ...
 
-        <a class="ui labeled icon button green" href="{{  path('webgriffe_sylius_akeneo_product_enqueue', {'productId': product.id }) }}" >  
+        <a class="ui labeled icon button violet" href="{{  path('webgriffe_sylius_akeneo_product_enqueue', {'productId': product.id }) }}" >  
```

Comment:
This is not related to this PR. Moreover I think we used `purple` and not `violet` before.

Reply:
Yes, I will do it in master directly or open a proper issue. The class is violet not purple, see [here](https://github.com/webgriffe/SyliusAkeneoPlugin/blob/aa1406faa3d281f15594db8d1df09988862ef0ef/src/Resources/config/config.yaml#L66).